db-scripts/src/main/resources/migration.sql

```diff
@@ -380,6 +380,8 @@ ALTER TABLE `mutation` ADD COLUMN `DRIVER_FILTER_ANNOTATION` VARCHAR(80) NULL;
 ALTER TABLE `mutation` ADD COLUMN `DRIVER_TIERS_FILTER` VARCHAR(50) NULL;
 ALTER TABLE `mutation` ADD COLUMN `DRIVER_TIERS_FILTER_ANNOTATION` VARCHAR(80) NULL;
 
+UPDATE info SET DB_SCHEMA_VERSION="2.4.0";
```

Comment:
was 2.4.0 missing?

Reply:
@inodb, added back on in this PR. was accidentally chopped off.